PFR_solver_CANTERA.m

```diff
@@ -0,0 +1,43 @@
+function [F] = PFR_solver_CANTERA(x,initial,gas,mdot,A_in,dAdx,k)
+
+rho = initial(1);
+T = initial(2);
+Y = initial(3:length(initial));
+
+if k==1
+    A = A_in-k*dAdx*x;
+elseif k==-1|k==0

```

Comment:
My linter suggests using `||` for this comparison. However, if `k = 1`, doesn't this result in the area decreasing anyways? I think you have a logic error here.

Reply:
Good catch. I fixed it